### ISSUE TITLE
disable GIF / emoji autoplay accessibility feature

### ISSUE THREAD

**Issue by ryleu** [stoatchat/for-web#664]
i would like to see the option for the accessibility feature where GIFs and emoji do not autoplay unless they are hovered over

i find it very distracting to have GIFs autoplay and others may have epilepsy issues due to autoplay

**Comment by without-ordinary** [comment]
This is a basic accessibility requirement. I cannot even look at any of my public chats without getting nauseous from the occasional animated thing that may happen to be in the chat history.

**Comment by LeopoldTal** [comment]
I've noticed always-active animations on:

Animated images:

- animated gifs in messages
- avatars in the server members list (right-hand panel)
- server banners in the "Discover" list
- server banners at the top of the channel list

Animated elements:

- scrolling when opening a channel with lots of message history
- fadein when opening settings
- fadein when opening a lightbox

This is very difficult to work around with userscripts/userstyles on my end, so this is making it impossible for me to use Stoat without just hiding all images.

I need to be able to disable those animations, either via a setting or via `prefers-reduced-motion: reduce`. Being able to play animated images on interaction would be ideal, but not strictly needed for accessibility.

Additionally, avatars animate on message hover, e.g. when using the context menu to reply etc. to the message. Could the target be made much smaller (just the avatar itself, maybe also the username)?

**Comment by prismflasher** [comment]
I don't think "prefers-reduced-motion" is an acceptable toggle for this, it should be its own discrete option, perhaps defaulting to match the detected reduced motion setting when the client is opened for the first time after the setting is added. My reasoning is that I've sometimes had trouble getting my OS to properly report me having enabled this setting to begin with, and I'd be very unhappy if I couldn't enable an accessibility feature I need due to a software bug when it's completely avoidable.

**Comment by ryleu** [comment]
![image](https://github.com/user-attachments/assets/e538c369-040f-4c73-b767-db00b28903ec)

discord's approach to this works well and i think it does not make sense to reinvent the wheel here
